common/test/acceptance/pages/lms/oauth2_confirmation.py

```diff
@@ -0,0 +1,53 @@
+"""
+Pages relevant for OAuth2 confirmation.
+"""
+from . import BASE_URL
```

Comment:
Done. No other acceptance test uses absolute imports for these things. If this is something you care about, you might consider filing a ticket for it.